### PR TITLE
Helm charts

### PR DESCRIPTION
**Pull request by KarolJastrzebski**

### CODE REVIEW

File: charts/coh-cor/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dependencies:
+  - name: postgresql
+    version: ~2.6.5
+    repository: '@stable'
+  - name: java
+    version: 0.0.11
```
Comment (KevinJCross) [inline charts/coh-cor/requirements.yaml]: you need to update this to ~0.0.12 .... this stops the process being run as root.

Comment (KarolJastrzebski) [reply charts/coh-cor/requirements.yaml]: Thanks, done it.

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,13 @@
-FROM openjdk:8-jre-alpine
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.2
```
Comment (timwebster9) [inline Dockerfile]: can you use this image instead (it's the latest): `hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0`

Comment (KarolJastrzebski) [reply Dockerfile]: Thanks, done.

**Comment by KarolJastrzebski** [comment]
Functional tests use JPA calls to prepare, assert, and/or clean up data in the database. This pull-request is put on hold until tests are refactored to use HTTP-based API instead, or postgresql service is exposed to the CI.

**Pull request closed without merging**